recipe/meta.yaml

```diff
@@ -1,35 +1,35 @@
+{% set name = "twine" %}
 {% set version = "3.7.1" %}
```

Comment:
why not 4.0.2 ?

Reply:
I do agree with you, and I will do it (here https://anaconda.atlassian.net/browse/PKG-2524). But this version was in the making for a long and it's ready. I would just build it and upload it.

Reply:
is it ok to merge this one @cbouss ?

Reply:
that's great! thanks, will merge this